packages/core/core/src/resolveOptions.js

```diff
@@ -104,6 +104,19 @@ export default async function resolveOptions(
       : undefined;
 
   let shouldBuildLazily = initialOptions.shouldBuildLazily ?? false;
+  let lazyIncludes = initialOptions.lazyIncludes ?? [];
+  if (lazyIncludes.length > 0 && !shouldBuildLazily) {
```

Comment:
After I created this PR I questioned as to whether having either `--lazy-includes` or `--lazy-excludes` should automatically enable lazy compilation?

That is - currently you have to do `--lazy --lazy-includes "src/routes/**"`, should it be enough to just do `--lazy-includes "src/routes/**"`? My original thinking was "--lazy turns on the feature, --lazy-(ex|in)cludes configures it).